docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: docs
+title: Browsers
+permalink: /documentation/concepts/browsers.html
+---
+# Browsers
+
+This topic lists browsers supported by TestCafe and describes how to use various browser features.
+
+* [Browser Support](#browser-support)
+* [Test in Headless Mode](#test-in-headless-mode)
+* [Use Chromium Device Emulation](#use-chromium-device-emulation)
+* [User Profiles](#user-profiles)
+
+## Browser Support
+
+### Officially Supported Browsers
+
+While TestCafe is designed to support most modern browsers, there are a limited number
+of *officially supported* browsers against which TestCafe is actively tested.
+
+* Google Chrome: Stable, Beta, Dev and Canary
+* Internet Explorer (11+)
+* Microsoft Edge (legacy and Chromium-based)
+* Mozilla Firefox
+* Safari
+* Google Chrome mobile
+* Safari mobile
+
+> TestCafe supports the latest version of each browser unless specified explicitly.
+
+### Locally Installed Browsers
+
+TestCafe can automatically detect popular browsers installed on the local computer.
+You can use a short name - *browser alias* - to identify these browsers when you launch tests.
```

Comment:
Can we say:
You can use a short name - the *browser alias* - to identify browsers when you launch a test.